usr/sbin/so-elastic-start

```diff
@@ -49,10 +43,14 @@ if [ "$DOMAIN_STATS_ENABLED" = "yes" ]; then
 	fi
 fi
 
-# ELASTICSEARCH
-# Publish ports to localhost 127.0.0.1 ONLY
 if [ "$ELASTICSEARCH_ENABLED" = "yes" ]; then
 	echo -n "so-elasticsearch: "
+	
+	# Configure CapMe with correct host:port pair
+        CAPMECONF="/var/www/so/capme/.inc/config.php"
```

Comment:
Do we need to duplicate these parameters into another configuration file?  Can CapMe not read from /etc/nsm/securityonion.conf?

Reply:
Not sure if it was the way I was implementing it, but I tried both in `callback-elastic.php` and `config.php` to do the following:

$elastic_host = shell_exec('grep ELASTICSEARCH_HOST /etc/nsm/securityonion.conf | cut -d= -f2 | sed "s/\"//g"');
$elastic_port = shell_exec('grep ELASTICSEARCH_PORT /etc/nsm/securityonion.conf | cut -d= -f2 | sed "s/\"//g"');

...but for some reason, could not get the variables passed correctly to the shell_exec line executing the query again ES, hence I fell back to `so-elastic-start` setting the values in `config.php` and simply reading that in into `callback-elastic.php`, which worked fine.

Reply:
Keep in mind, the above worked fine in `config.php` (for example if I ran `php -f config.php` in cli, echoing the output), but the values were not correctly read in to `callback-elastic-.php`.